### ISSUE TITLE
java.lang.NullPointerException in nl.xservices.plugins.accessor.AbstractCalendarAccessor.getActiveCalendars

### ISSUE THREAD

**Issue by tuberlinapp** [EddyVerbruggen/Calendar-PhoneGap-Plugin#289]
Android-Version: Android 5.0
Device: Galaxy TabS 10.5 (chagallwifi) 1 
```
java.lang.NullPointerException: Attempt to invoke interface method 'boolean android.database.Cursor.moveToFirst()' on a null object reference
    at nl.xservices.plugins.accessor.AbstractCalendarAccessor.getActiveCalendars(AbstractCalendarAccessor.java:266)
    at nl.xservices.plugins.Calendar$3.run(Calendar.java:224)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
    at java.lang.Thread.run(Thread.java:818)
```

**Comment by EddyVerbruggen** [comment]
Hmm, so it seems like that device doesn't have any active calendars -- is that correct?

**Comment by tuberlinapp** [comment]
I don't have any additional information, 'cause we got the crash-information from the Google store and we don't know the user, sorry.

**Comment by EddyVerbruggen** [comment]
No worries. I just assume it's the case and add a check to the code. Thanks for reporting it!

**Comment by EddyVerbruggen** [comment]
Fixed in the upcoming 4.3.8 release and available now from the master repo.
